crates/sncast/tests/integration/wait_for_tx.rs

```diff
@@ -50,6 +52,29 @@ async fn test_rejected_transaction() {
     assert!(resp.to_string().contains("InsufficientMaxFee"));
 }
 
+#[tokio::test]
+#[should_panic(expected = "Transaction has been reverted = Insufficient max")]
```

Comment:
Are you sure the revert message is `Insufficient max` ? seems a bit odd

Reply:
In this example yes! The max_fee is not enough, so that's the reason for the revert

Reply:
I just checked and it seems the message is actually longer :P I guess should_panic matches pattern somehow. Let's inlude a bit longer message to avoid confusion (`(...) max fee:` should be enough)

```
---- integration::wait_for_tx::test_wait_for_reverted_transaction stdout ----
Transaction hash = 0x2e7a07b5273b142f9657bac3f642eb5f9bb38512545f6b564791d2cbbc91203
thread 'integration::wait_for_tx::test_wait_for_reverted_transaction' panicked at crates/sncast/tests/integration/wait_for_tx.rs:74:10:
called `Result::unwrap()` on an `Err` value: Transaction has been reverted = Insufficient max fee: max fee: Fee(332323232324342), actual fee: Fee(801700000000000)
```